packages/playground/vue/src-import/template.html

```diff
@@ -1,3 +1,5 @@
 <h2>SFC Src Imports</h2>
 <div class="src-imports-script">{{ msg }}</div>
 <div class="src-imports-style">This should be tan</div>
+<a></a>
```

Comment:
Please use a multi-word name here. `a` is a native HTML tag.

Reply:
Oh damn, you are right :smile: